File: autoburn.pc/autoburn/msghandler/RxMsgHandlerInfo.cs
using Autoburn.Manager;
using Autoburn.Net;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Autoburn.MsgHandler
{
    class RxMsgHandlerInfo:RxMsgHandlerBase
    {
        public RxMsgHandlerInfo(DeviceManager d) : base(d, MSG_TYPE_INFO)
        {
        }
        public override void Process()
        {
            base.Process();
            if (string.IsNullOrEmpty(Msg))
            {
                return;
            }


        }
    }
}

File: autoburn.android/app/src/main/java/com/autoburn/msghandler/RxMsgHandlerInfo.java
package com.autoburn.msghandler;

import android.util.Log;

/**
 * Created by chowhsienwu on 2017/5/4.
 */
public class RxMsgHandlerInfo extends RxMsgHandlerBase {
    public RxMsgHandlerInfo(){
        super(MSG_TYPE_INFO);
    }

    @Override
    protected Void doInBackground(String... params) {
        Msg = params[0];

        Log.i(TAG, " the Msg is " + Msg);
        return null;
    }
}

